From d4ed23af4125f4cbe781f6d78584f32dacc725fc Mon Sep 17 00:00:00 2001
From: Simone Schwizer <simone.schwizer@bytesatwork.ch>
Date: Tue, 30 Sep 2025 11:34:31 +0200
Subject: [PATCH] configs: am62x_bytedevkit: Add quiet to kernel bootargs

Reduces output during boot to speed up total boot time.

Signed-off-by: Simone Schwizer <simone.schwizer@bytesatwork.ch>
---
 include/configs/am62x_bytedevkit.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/am62x_bytedevkit.h b/include/configs/am62x_bytedevkit.h
index 42e77a8972a..081390b74fa 100644
--- a/include/configs/am62x_bytedevkit.h
+++ b/include/configs/am62x_bytedevkit.h
@@ -34,7 +34,7 @@
 	"args_generic=setenv bootargs console=${console} " \
 		"earlycon=ns16550a,mmio32,0x02800000 ${mtdparts} " \
 		"root=/dev/mmcblk${mmc_dev}p${mmc_part} " \
-		"rootfstype=ext4 rootwait ${bootargs_append}; " \
+		"rootfstype=ext4 rootwait quiet ${bootargs_append}; " \
 	"\0" \
 	"dtb_fixups=fdt addr ${dtbaddr}; fdt resize 0x10000; " \
 		"setexpr second_mac gsub ':' ' ' $eth1addr; " \
